dataland-frontend/tests/e2e/utils/ApiUtils.ts

```diff
@@ -24,3 +24,15 @@ export async function countCompanyAndDataIds(
     matchingCompanies,
   };
 }
+
+export function interceptAllAndCheckFor500Errors(): void {
+  cy.intercept("/api/**", (req) => {
+    const allow500 = req.headers["DATALAND-ALLOW-5XX"] === "true";
+    delete req.headers["DATALAND-ALLOW-5XX"];
+    req.continue((res) => {
+      if (res.statusCode >= 500 && !allow500) {
```

Comment:
Is checking for 500 enough or is any 5xx required to be intercepted?

Reply:
IMO any 5XX error hints at a server-side error that should not occur during normal operation and should therefore fail the tests (if not explicitly allowed)

Reply:
I swear there was a == yesterday.